app/views/submissions/show.erb

```diff
@@ -6,6 +6,9 @@
     </h1>
 
     <div class="pull-right submission-toolbar hidden-xs">
+      <% if submission.user_exercise.help_requested? %>
+        <span class="label label-danger">Requested help</span>
```

Comment:
Really minor thing: 

I'd use a different class than 'label-danger' (maybe 'label-help-requested') and then duplicate the label-danger css as label-help-requested. 

This is because they are different concepts, that just happen to be currently look same. 
If the colour scheme were to change in the future, (for example, to a blueish theme.) we might want to make the help requested labels blue while still keeping anything that is actually dangerous red.